src/common/vector_operations/comparison_operators.cpp

```diff
@@ -182,9 +182,9 @@ static void NestedComparisonExecutor(Vector &left, Vector &right, Vector &result
 	if (!leftv.validity.AllValid() || !rightv.validity.AllValid()) {
 		ComparesNotNull(leftv, rightv, result_validity, count);
 	}
-	ValidityMask copy;
-	copy.SetAllValid(count);
-	copy.Copy(result_validity, count);
+	ValidityMask original_mask;
```

Comment:
This reads to me like all that changed was renaming `copy` to `original_mask`... what am I missing?